src/components/GalleryMedia.js

```diff
@@ -25,20 +26,25 @@ function GalleryMedia({
   const altName = media.title || media.description || ''
   let caption = media.description || media.title || ''
   let type = 'image'
+  let customClass = null
   const [showMedia, setShowMedia] = useState(false)
   const imgRef = useRef()
-  let imgCount = (index + 1).toString() + " / " + mediaCount.toString();
+  const imgCount = (index + 1).toString() + ' / ' + mediaCount.toString()
   if (media?.type?.includes('video')) {
     modelLink = null
     type = 'video'
+    customClass = styles.mediaVideo
     iframeSrc = getVideoURL(media)
   } else if (media?.type?.includes('walkthrough')) {
     modelLink = null
     type = 'walkthrough'
     iframeSrc = getWalkthroughURL(media)
+    customClass = styles.mediaVideo
```

Comment:
Maybe this should only get applied if there is a caption and showCaption is true?